entrypoint.sh

```diff
@@ -133,7 +133,10 @@ fi
 
 
 if [[ ${_RUN_AS_ROOT} == "true" ]]; then
-  [[ $(id -u) -eq 0 ]] && ( "$@" ) || ( echo "ERROR: RUN_AS_ROOT env var is set to true but the user has been overriden and is not running as root"; exit 1 )
+  if [[ $(id -u) -ne 0 ]]; then
```

Comment:
youve lost the part where if RUN_AS_ROOT is true and id -u is 0 that it still needs to run $@